workflowState.yaml

```diff
@@ -57,6 +74,54 @@ components:
             $ref: '#/components/schemas/KeyValue'
         commandRequest:
           $ref: '#/components/schemas/CommandRequest'
+    WorkflowStateDecideRequest:
+      type: object
+      properties:
+        workflowType:
+          type: string
+        workflowStateId:
+          type: string
+        searchAttributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/KeyValue'
+        queryAttributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/KeyValue'
+        commandRequest:
+          $ref: '#/components/schemas/CommandRequest'
+    WorkflowStateDecideResponse:
+      type: object
+      properties:
+        stateDecision:
+          type: object
+          items:
+            $ref: '#/components/schemas/StateDecision'
+    StateDecision:
+      type: object
+      properties:
+        waitForMoreCommandResults:
+          type: boolean
+        nextStates:
+          type: array
+          items:
+            $ref: '#/components/schemas/StateMovement'
+        upsertSearchAttributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/KeyValue'
+        upsertQueryAttributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/KeyValue'
+    StateMovement:
+      type: object
+      properties:
+        stateId:
+          type: string
+        nextStateInput:
```

Comment:
@longquanzheng is this type correct?  I see activity command input in example is of Object type, and in this idl it's string type, so here I'm doing the same

Reply:
Yes, I think we have to use string here to represent arbitrary object which is serialized into string (json , base 64 etc) because swagger/REST doesn’t have a raw object type.